jiant/scripts/preproc/superglue_leaderboard_submission_formatter.py

```diff
@@ -0,0 +1,52 @@
+"""Translate raw prediction files for SuperGLUE tasks into format expected by SuperGLUE leaderboard.
+
+This script helps translate the raw prediction files for SuperGLUE tasks into the jsonl files
+required by the SuperGLUE leaderboard. See https://super.gluebenchmark.com/faq for leaderboard info.
+
+To use this script specify one or more raw prediction input filepath in raw_pred_input_filepaths.
+
+"""
+import json
+import torch
+
+from jiant.tasks import retrieval
+
+# for each SuperGLUE task, provide a filepath for the raw prediction file as a value in this map:
+raw_pred_input_filepaths = {
+    "boolq": None,
+    "cb": None,
+    "copa": None,
+    "mrc": None,
+    "record": None,
+    "rte": None,
+    "wic": None,
+    "wsc": None,
+    # "TBD1": None, # Broadcoverage diagnostic
+    # "TBD2": None, # Winogender Schema diagnostic
+}
```

Comment:
Possible to use tasks/constants.py SuperGLUE definition here but can wait until all tasks are implemented.

Reply:
Yup, done in edd392c.